src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannelFactory.cs

```diff
@@ -450,8 +482,7 @@ protected override void OnEndClose(IAsyncResult result)
 
             protected internal override Task OnCloseAsync(TimeSpan timeout)
```

Comment:
We now have multiple implementations of OnClose,we have the synchronous OnClose, the APM OnBeginClose and the Task based OnCloseAsync. In other places we've only got one implementation and the others call it.